### PR TITLE
chore: Windows build improvements

### PR DESCRIPTION
**Pull request by hakim0101**
I propose vcpkg way of build on Windows/MSVC.
Some minor adjustments in cmake and install instructions.

- - -
This change is [Reviewable](https://reviewable.io/reviews/TokTok/c-toxcore/2377)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -178,16 +178,33 @@ make install
 
 In addition to meeting the [requirements](#requirements), you need a version of Visual Studio (the [community edition](https://www.visualstudio.com/vs/visual-studio-express/) is enough) and a CMake version that's compatible with the Visual Studio version you're using.
 
-You must also ensure that the msvc versions of dependencies you're using are placed in the correct folders.
+With the dependencies you have two options:
 
-For libsodium that is `c-toxcore/third_party/libsodium`, and for pthreads-w32, it's `c-toxcore/third_party/pthreads-win32`
+  - bootstrap vcpkg as described here https://github.com/microsoft/vcpkg#quick-start-windows and then do
```
Comment (Tha14) [inline INSTALL.md:183]: The link should have this format(you can replace the text in [] with words):
``[https://github.com/microsoft/vcpkg#quick-start-windows](https://github.com/microsoft/vcpkg#quick-start-windows)```

File: INSTALL.md [hunk 2]
```diff
@@ -178,16 +178,33 @@ make install
 
 In addition to meeting the [requirements](#requirements), you need a version of Visual Studio (the [community edition](https://www.visualstudio.com/vs/visual-studio-express/) is enough) and a CMake version that's compatible with the Visual Studio version you're using.
 
-You must also ensure that the msvc versions of dependencies you're using are placed in the correct folders.
+With the dependencies you have two options:
```
Comment (Tha14) [inline INSTALL.md:181]: Just nitpicking here so don't take this the wrong way. I'd propose replacing "With" with "In regards to" or "Regarding".

Comment (hakim0101) [reply INSTALL.md]: Done!

File: INSTALL.md (hunk 1 above)
Comment (hakim0101) [reply INSTALL.md]: Done

**Review by iphydf — APPROVED** [review]

**Review by nurupo — CHANGES_REQUESTED** [review]

File: INSTALL.md [hunk 3]
```diff
@@ -178,16 +178,33 @@ make install
...
 cd _build
-cmake ..
+```
+then for the first way (vcpkg)
+```
+cmake .. -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]\vcpkg\scripts\buildsystems\vcpkg.cmake
+```
+and for the second way (manual dependencies resolution)
+```
+cmake .. -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]\vcpkg\scripts\buildsystems\vcpkg.cmake
```
[the remark this answers is not in the document]
Comment (iphydf) [reply INSTALL.md:204]: @hakim0101 ping.

**Comment by iphydf** [comment]
We now use vcpkg in cmake to find the packages automatically.

**Pull request closed without merging**